### DISCUSSION TITLE
Updated existing CHANGELOG.md file?

### DISCUSSION THREAD

**Discussion by ashmckenzie** [goreleaser/goreleaser#4661] [Ideas, Issue Triage, and General discussions]
Hi everyone 👋 

We're looking to automate the release process for a project where we currently create a GitLab MR to update a `CHANGELOG.md` file that's checked in as part of the workflow.  

Is anyone aware of any way to configure goreleaser to not only create builds, releases etc but also git update a file as part of it's execution?

Thanks in advance 🙂

**Comment by caarlos0** [comment]
hey, this is not possible (yet), but it's something I have in mind for future releases

**Reply by ashmckenzie** [reply]
Thanks @caarlos0 🙇 For GitHub I believe maybe I could get it to work with https://goreleaser.com/includes/prs. I'll investigate using hooks to see what's possible there 🙂

**Reply by caarlos0** [reply]
that page shouldn't be served standalone...

its included in `brew`, `nix`, and others. It'll not work

**Comment by caarlos0** [comment]
centralizing in https://github.com/orgs/goreleaser/discussions/5965
